lib/active_model/serializable_resource.rb

```diff
@@ -30,7 +38,11 @@ def serialization_scope_name=(scope_name)
     end
 
     def adapter
-      @adapter ||= ActiveModel::Serializer::Adapter.create(serializer_instance, adapter_opts)
+      @adapter ||=
+        begin
```

Comment:
why is this wrapped in a begin/end? if it's needed, could a comment be added explaining why, and what to do if an exception is raised?